pkg/expr/convert_to_long_test.go

```diff
@@ -0,0 +1,48 @@
+package expr
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConvertNumericMultiToLong(t *testing.T) {
+	input := data.Frames{
+		data.NewFrame("test",
+			data.NewField("Value", data.Labels{"city": "MIA"}, []int64{5})),
+		data.NewFrame("test",
+			data.NewField("Value", data.Labels{"city": "LGA"}, []int64{7}),
+		),
+	}
+	expectedFrame := data.NewFrame("",
```

Comment:
```suggestion
	expectedFrame := data.NewFrame("test",
```
I tried this change locally, and it fails. ie. We would need to update the application code.

I feel quite strongly that we should preserve the Frame name when converting, rather than discarding it. The significance of this is that:
- If we received an array (a numeric-multi) containing frames which belonged to two different frame names, we'd output two different NumericLong frames.

This feels the right thing to do to me.

Reply:
The name of the frame has no special meaning in these types (according to the dataplane doc). The idea is to collapse the input into a single table so it maps to a SQL table (Which is what Wide/Multi - Long conversion does).